manifests/generated/operator-csv.yaml.in

```diff
@@ -685,6 +685,8 @@ spec:
                 env:
                 - name: OPERATOR_IMAGE
                   value: {{.DockerPrefix}}/virt-operator{{if .VirtOperatorSha}}@{{.VirtOperatorSha}}{{else}}:{{.DockerTag}}{{end}}
+                - name: HCO_KV_IO_VERSION
+                  value {{if .hcoKvIoVersion}}{{.hcoKvIoVersion}}{{end}}
```

Comment:
hcoKvIoVersion should be `HcoKvIoVersion` with a uppercase 'H'. Then i think this will work as expected.